tools/triageHelper/function_app.py

```diff
@@ -0,0 +1,67 @@
+import azure.functions as func
+import logging
+import requests
+import urllib.parse
+
+app = func.FunctionApp(http_auth_level=func.AuthLevel.ANONYMOUS)
+
+REPOS = [
+    "Azure/azure-functions-durable-extension",
+    "Azure/durabletask",
+    "Azure/azure-functions-durable-python",
+    "microsoft/durabletask-java",
+    "microsoft/durabletask-dotnet",
+    "microsoft/durabletask-mssql",
+    "microsoft/durabletask-netherite",
+    "microsoft/DurableFunctionsMonitor"
```

Comment:
the PS SDK is notably missing from here. That's because it uses a slightly different "triage" label.